cluster-up/cluster/kind-k8s-sriov-1.17.0/deploy_sriov_operator.sh

```diff
@@ -0,0 +1,254 @@
+  operator_path=${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/sriov-network-operator-${OPERATOR_GIT_HASH}
+  if [ ! -d $operator_path ]; then
+    curl -LSs https://github.com/openshift/sriov-network-operator/archive/${OPERATOR_GIT_HASH}/sriov-network-operator.tar.gz | tar xz -C ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/
+  fi
+
+  echo 'Installing the SR-IOV operator'
+  pushd $operator_path
+    export RELEASE_VERSION=4.4
+    export SRIOV_NETWORK_OPERATOR_IMAGE=quay.io/openshift/origin-sriov-network-operator:${RELEASE_VERSION}
+    export SRIOV_NETWORK_CONFIG_DAEMON_IMAGE=quay.io/openshift/origin-sriov-network-config-daemon:${RELEASE_VERSION}
+    export SRIOV_NETWORK_WEBHOOK_IMAGE=quay.io/openshift/origin-sriov-network-webhook:${RELEASE_VERSION}
+    export NETWORK_RESOURCES_INJECTOR_IMAGE=quay.io/openshift/origin-sriov-dp-admission-controller:${RELEASE_VERSION}
+    export SRIOV_CNI_IMAGE=quay.io/openshift/origin-sriov-cni:${RELEASE_VERSION}
+    export SRIOV_DEVICE_PLUGIN_IMAGE=quay.io/openshift/origin-sriov-network-device-plugin:${RELEASE_VERSION}
+    export OPERATOR_EXEC=${KUBECTL}
+    make deploy-setup-k8s SHELL=/bin/bash  # on prow nodes the default shell is dash and some commands are not working
+  popd
+
+  echo 'Generating webhook certificates for the SR-IOV operator webhooks'
+  pushd "${CERTCREATOR_PATH}"
+    go run . -namespace sriov-network-operator -secret operator-webhook-service -hook operator-webhook -kubeconfig $KUBECONFIG_PATH
+    go run . -namespace sriov-network-operator -secret network-resources-injector-secret -hook network-resources-injector -kubeconfig $KUBECONFIG_PATH
+  popd
+
+  echo 'Setting caBundle for SR-IOV webhooks'
+  wait_k8s_object "validatingwebhookconfiguration" "operator-webhook-config"
+  _kubectl patch validatingwebhookconfiguration operator-webhook-config --patch '{"webhooks":[{"name":"operator-webhook.sriovnetwork.openshift.io", "clientConfig": { "caBundle": "'"$(cat $CERTCREATOR_PATH/operator-webhook.cert)"'" }}]}'
+
+  wait_k8s_object "mutatingwebhookconfiguration"   "operator-webhook-config"
+  _kubectl patch mutatingwebhookconfiguration operator-webhook-config --patch '{"webhooks":[{"name":"operator-webhook.sriovnetwork.openshift.io", "clientConfig": { "caBundle": "'"$(cat $CERTCREATOR_PATH/operator-webhook.cert)"'" }}]}'
+
+  wait_k8s_object "mutatingwebhookconfiguration"   "network-resources-injector-config"
+  _kubectl patch mutatingwebhookconfiguration network-resources-injector-config --patch '{"webhooks":[{"name":"network-resources-injector-config.k8s.io", "clientConfig": { "caBundle": "'"$(cat $CERTCREATOR_PATH/network-resources-injector.cert)"'" }}]}'
+
+  return 0
+}
+
+function apply_sriov_node_policy {
+  local -r policy_file=$1
+  local -r pfs_names_json_arrays=$2
+  local -r num_vfs=$3
+  
+  # Format to singal array ["ens1","ens2"],["ens3"] => ["ens1","ens2","ens3"]
+  local all_pfs_names_json_array=$(sed 's/\].*\[/"," /g' <<< "$pfs_names_json_arrays")
```

Comment:
Please use envsubst as we did before, i find sed here too complex and not maintainable.